### ISSUE TITLE
Importmap:install is failling with my setup. Seems related to recent change PR #58875

### ISSUE THREAD

**Issue by nuncanada** [symfony/symfony#59303]
### Symfony version(s) affected

7.2.0

### Description

Error:

PS C:\Users\flaviobb\Desktop\ProjetosContratos-pncp\contratos_symfony> composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run composer update or composer update <package name>.
Nothing to install, update or remove
Package league/uri-parser is abandoned, you should avoid using it. Use league/uri-interfaces instead.
Generating autoload files
129 packages you are using are looking for funding.
Use the composer fund command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

Executing script cache:clear [OK]
Executing script importmap:install [KO]
[KO]
Script importmap:install returned with error code 1
!! 0/21 >--------------------------- https://cdn.jsdelivr.net/npm/jquery-ui@1.13.2/+esm
!! In ErrorChunk.php line 73:
!!
!! Body size limit exceeded
!!
!!
!! In Http2ConnectionProcessor.php line 711:
!!
!! Body size limit exceeded
!!
!!
!! In Http2ConnectionProcessor.php line 757:
!!
!! Body size limit exceeded
!!
!!
!! importmap:install
!!
!!
Script @auto-scripts was called via post-install-cmd

Once I changed the line in AmpHttpClient.php from
$request->setBodySizeLimit(0);
To
$request->setBodySizeLimit(10000000000000000);

Everything worked just fine!!!

### How to reproduce

My importmap:

```
<?php

/**
 * Returns the importmap for this application.
 *
 * - "path" is a path inside the asset mapper system. Use the
 *     "debug:asset-map" command to see the full list of paths.
 *
 * - "entrypoint" (JavaScript only) set to true for any module that will
 *     be used as an "entrypoint" (and passed to the importmap() Twig function).
 *
 * The "importmap:require" command can be used to add new entries to this file.
 */
return [
    'jquery-ui' => [
        'version' => '1.13.2',
    ],
    'jquery' => [
        'version' => '3.7.1',
    ],
    'jquery/dist/jquery.min.js' => [
        'version' => '3.7.1',
    ],
    'bootbox' => [
        'version' => '6.0.0',
    ],
    'ckeditor' => [
        'version' => '4.12.1',
    ],
    '@fortawesome/fontawesome-free' => [
        'version' => '6.5.2',
    ],
    '@fortawesome/fontawesome-free/css/fontawesome.min.css' => [
        'version' => '6.5.2',
        'type' => 'css',
    ],
    '@fortawesome/fontawesome-svg-core' => [
        'version' => '6.5.2',
    ],
    '@fortawesome/fontawesome-svg-core/styles.min.css' => [
        'version' => '6.5.2',
        'type' => 'css',
    ],
    '@fortawesome/free-brands-svg-icons' => [
        'version' => '6.5.2',
    ],
    '@fortawesome/free-regular-svg-icons' => [
        'version' => '6.5.2',
    ],
    '@fortawesome/free-solid-svg-icons' => [
        'version' => '6.5.2',
    ],
    'bootstrap' => [
        'version' => '5.3.3',
    ],
    '@popperjs/core' => [
        'version' => '2.11.8',
    ],
    'bootstrap/dist/css/bootstrap.min.css' => [
        'version' => '5.3.3',
        'type' => 'css',
    ],
    'bootstrap5-autocomplete' => [
        'version' => '1.1.26',
    ],
    'bootstrap5-autocomplete/autocomplete.js' => [
        'version' => '1.1.26',
    ],
    
    'app' => [
        'path' => './assets/app.js',
        'entrypoint' => true,
    ],
    '@hotwired/stimulus' => [
        'version' => '3.2.2',
    ],
    'tom-select' => [
        'version' => '2.3.1',
    ],
    'tom-select/dist/css/tom-select.default.css' => [
        'version' => '2.3.1',
        'type' => 'css',
    ],
    'tom-select/dist/css/tom-select.bootstrap5.css' => [
        'version' => '2.3.1',
        'type' => 'css',
    ],
    '@stm/admin-bundle' => [
        'path' => './vendor/stm/admin-bundle/assets/StmAdmin.js',
    ],
];
```

### Possible Solution

It seems the code in importmap:install is using BodySizeLimit to test if the response has overflown, but with the change to 0 the test is failing because every response has size > 0...

**Comment by nuncanada** [comment]
if ($stream->received >= $stream->request->getBodySizeLimit()) {
            $this->handleStreamException(new Http2StreamException(
                "Body size limit exceeded",
                $streamId,
                Http2Parser::CANCEL
            ));

            return;
        }
        
        
        Line 759 in Http2ConnectionProcessor.php

**Comment by xabbuh** [comment]
Can you create a small example application that allows to reproduce your issue?

**Comment by xabbuh** [comment]
Okay, after investigating you need to update amphp/http-client. This was fixed there in amphp/http-client#297.
